src/main/java/frc/robot/subsystems/led/Leds.java

```diff
@@ -66,8 +71,22 @@ public enum LedMode {
   private static final double waveAllianceDuration = 2.0;
   private static final double autoFadeTime = 2.5; // 3s nominal
   private static final double autoFadeMaxTime = 5.0; // Return to normal
+  private static final double noteAngle = 5.0;
+
+  private NetworkTable ledTable;
+  private NetworkTableEntry ledModeEntry;
+  private NetworkTableEntry ledTestingEntry;
+  private NetworkTableEntry ledColor;
```

Comment:
Right now ledColor is only set in a couple specific places. Can you either update it automatically from each pattern method (e.g. wave(), solid, etc) or just remove it?